.changeset/brown-turtles-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': patch
```

Comment:
```suggestion
'@itwin/itwinui-css': patch
'@itwin/itwinui-react': patch
```

Reply:
Ohh, to confirm, since this is a CSS only change, do we need to add `itwinui-react`?

Reply:
yes, because itwinui-react produces its own stylesheet